model/testing/src/icon4py/model/testing/serialbox.py

```diff
@@ -1244,6 +1244,7 @@ def z_theta_v_fl_e(self):
         return self._get_field("z_theta_v_fl_e", dims.EdgeDim, dims.KDim)
 
 
+# TODO (magdalena) rename?
```

Comment:
I have no real good name for this. it contains fields which are serialized at the end of `nh_solve` after the predictor and corrector steps run, where the do some computation to avoid halo exchange.